src/_angular/transition-setting/transition-setting.html

```diff
@@ -0,0 +1,102 @@
+        <option value="page">{{"transition.by.page" | translate}}</option>
+      </select>
+    </div>
+  </div>
+</div>
+<div class="row" ng-if="transition.by === 'fade'">
+  <div class="col-md-3">
+    <div class="form-group">
+      <label class="control-label">{{"transition.duration" | translate }}</label>
+      <div class="input-group">
+        <input name="transition-duration" type="number"  class="form-control" ng-model="transition.duration">
+        <span class="input-group-addon">sec</span>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="row" ng-if="transition.by === 'scroll'">
+  <div class="col-md-3">
+    <div class="form-group">
+      <label class="control-label">{{"transition.duration" | translate }}</label>
+      <div class="input-group">
+        <input name="transition-duration" type="number"  class="form-control" ng-model="transition.duration">
+        <span class="input-group-addon">sec</span>
+      </div>
+    </div>
+  </div>
+  <div class="col-md-3">
+    <div class="form-group">
+      <label class="control-label">{{"transition.speed.label" | translate}}</label>
+      <select name="transition-speed" ng-model="transition.speed" class="form-control">
+        <option value="slowest">{{"transition.speed.slowest" | translate}}</option>
+        <option value="slow">{{"transition.speed.slow" | translate}}</option>
+        <option value="medium">{{"transition.speed.medium" | translate}}</option>
+        <option value="fast">{{"transition.speed.fast" | translate}}</option>
+        <option value="fastest">{{"transition.speed.fastest" | translate}}</option>
+      </select>
+    </div>
+  </div>
+  <div class="col-md-3">
+    <div class="form-group">
+      <label class="control-label">{{"transition.resume.label" | translate}}</label>
+      <span popover="{{'transition.resume.tooltip' | translate}}" popover-trigger="click"
+            popover-placement="right" rv-tooltip></span>
+      <div class="input-group">
+        <input name="transition-resume" type="number" ng-model="transition.resume" class="form-control" />
+        <span class="input-group-addon">{{"common.units.seconds" | translate}}</span>
+      </div>
+    </div>
+  </div>
+  <div class="col-md-3">
+    <div class="form-group">
+      <label class="control-label">{{"transition.pud.label" | translate}}</label>
+      <span popover="{{'transition.pud.tooltip' | translate}}" popover-trigger="click"
+            popover-placement="right" rv-tooltip></span>
+      <div class="input-group">
+        <input name="transition-pud" type="number" ng-model="transition.pud" class="form-control" />
+        <span class="input-group-addon">{{"common.units.seconds" | translate}}</span>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="row" ng-if="transition.by === 'page'">
+  <div class="col-md-3">
+    <div class="form-group">
+      <label class="control-label">{{"transition.duration" | translate }}</label>
```

Comment:
@stulees Instead of duplicating some of the HTML, would it be possible to move the `ng-if` to the child elements instead?